crates/aptos-telemetry-service/src/types/auth.rs

```diff
@@ -25,7 +25,7 @@ pub struct AuthResponse {
     pub handshake_msg: Vec<u8>,
 }
 
-#[derive(Debug, Serialize, Deserialize, PartialEq, Eq)]
+#[derive(Debug, Serialize, Deserialize, PartialEq, Eq, Clone)]
```

Comment:
Do we need to clone the claim? Can we not simply move into the cache?

Reply:
I tried that before adding `clone`. I got this error. 
<img width="918" alt="Screenshot 2023-05-10 at 12 42 43 PM" src="https://github.com/aptos-labs/aptos-core/assets/7092734/958296dd-01ba-45b9-9f96-2efa03f43dce">